dev/watch.js

```diff
@@ -44,16 +44,21 @@ webpackBundler.watch(
             return browserSync.init(bsConfig);
         }
 
+        const info = stats.toJson();
         // send editing errors to console and browser
         if (stats.hasErrors() || stats.hasWarnings()) {
-            console.log(chalk.red(stats.toString('errors-only')));
+            console.log(info.errors);
```

Comment:
Will `info.errors` be defined if we enter this block because `stats.hasErrors()` is falsey? 

And if we do want log errors should we use `chalk.red`?

Reply:
no you're right, missed that - ta